src/current/v23.1/cockroach-statement-diag.md

```diff
@@ -9,7 +9,7 @@ Use the `cockroach statement-diag` [command]({% link {{ page.version.version }}/
 
 ## Required privileges
 
-Only members of the `admin` role can run `cockroach statement-diag`. By default, the `root` user belongs to the `admin` role.
+Only members of the `admin` role and users with the `VIEWSYSTEMTABLE` [system privilege]({% link v23.2/security-reference/authorization.md %}#privileges), which provides read-only access to system tables, can run `cockroach statement-diag`. By default, the `root` user belongs to the `admin` role.
```

Comment:
nit: this is linking to v23.2 docs; is that intentional?

Reply:
Done